src/xcb_ffi/mod.rs

```diff
@@ -427,25 +419,6 @@ impl RequestConnection for XCBConnection {
         }
     }
 
-    fn check_for_error(
-        &self,
-        sequence: SequenceNumber,
-    ) -> Result<Option<GenericError>, ConnectionError> {
-        let cookie = raw_ffi::xcb_void_cookie_t {
-            sequence: sequence as _,
-        };
-        let error = unsafe { raw_ffi::xcb_request_check(self.conn.0.as_ptr(), cookie) };
-        if error.is_null() {
-            Ok(None)
-        } else {
-            unsafe {
-                Ok(Some(GenericError::new(XCBConnection::wrap_error(
-                    error as _,
-                ))?))
-            }
-        }
-    }
-
```

Comment:
If you ask about it, I just reordered it to make it consistent with the order in the trait definition.